Makefile

```diff
@@ -62,4 +63,8 @@ push-staging:
 	make APP_ENV=staging build
 	docker-compose -f docker-compose.yml -f config/staging.yml up -d
 
-.PHONY: default build serve clean debug develop develop-rebuild dropdb test
+production: APP_ENV=production
+production: DOCKER_COMPOSE_FILE=docker-compose-production.yml
+production: serve-d
```

Comment:
Do you need to override this? Is it not possible to just override the `$(APP_ENV).yml` file?